docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.9"
+
+x-common-variables: &common-variables
+  POSTGRES_USER: ${POSTGRES_USER:-postgres}
+  POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres}
+  POSTGRES_DB: ${POSTGRES_DB:-template}
+
+services:
+  dbpg: 
+    container_name: postgres
+    image: postgres
+    environment:
+      <<: *common-variables
+      PGDATA: /data/postgres
+    ports:
+      - "5432:5432"
+    networks:
+      - backend 
+    restart: unless-stopped
```

Comment:
```suggestion
    restart: unless-stopped
    
    volumes:
      - kingdom-dbpg-data:/var/lib/postgresql/data/pgdata
```

Reply:
Do we need to create this directory locally?

Reply:
No, docker creates this for us